model/k8s/metrics.yaml

```diff
@@ -63,6 +63,21 @@ groups:
       - ref: network.interface.name
       - ref: network.io.direction
 
+  # k8s.container.* metrics
+  - id: metric.k8s.container.status.state
+    type: metric
+    metric_name: k8s.container.status.state
+    stability: experimental
+    brief: ""
+    instrument: updowncounter
+    unit: ""
+    attributes:
```

Comment:
I think we need to name these as `k8s.container.status.state` and `k8s.container.status.reason` so as to have meaningful names in case we want to re-use them in other signals like Entities (`k8s.container.reason` does not make a lot of sense).